renderer/configuration/README.md

```diff
@@ -0,0 +1,85 @@
+- **Station**: adding stations and their corresponding images
+
+### Example Format
+
+Here we describe the format of the JSON files under ```configuration```.
+
+```
+{
+    "version" = The current version of the configuration file
+
+    "player" = A list of player models that can be used in the game. Currently, robotouille is single-player ONLY. So the name of the player cannot be changed. 
+    {
+        "robot" = The name of the player (cannot be changed)
+        {
+            "front" = The image for the front of the player
+            "back" = The image for the back of the player
+            "left" = The image of the player facing left
+            "right" = The image of the player facing right
+        }
+    }
+
+    "floor" = The image for the floor
+
+    "item" = The configuration for items in the game
+    {
+        "constants" = A list of constants to be applied to all items
+        {
+            "STATION_ITEM_OFFSET" = How much higher or lower items should be rendered when placed on a station
+            "X_SCALE_FACTOR" = How much items should be scaled in the x direction
+            "Y_SCALE_FACTOR": How much items should be scaled in the y direction
+        }
+        "entities" = A list of items that can be used in the game
+        {
+            "item_name1" = The name of the item
+            {
+                "assets" = A list of images for the item
+                {
+                    "default" = The default image for the item
+                    "item_state1" = The image for an item if certain predicates are true
+                    {
+                        "asset" = The image for the item when the predicates is true
+                        "predicates" = A list of predicates that must be true for the image to be rendered
+                    }
+                    ...
+                }
+                "constants" = A list of constants to be applied to the item
+                {
+                    "STACK_OFFSET" = How much higher or lower the item should be rendered when stacked
+                }
+            }
+            ...
+        }
+    }
+    "station" = The configuration for stations in the game
+    {
+        "entities" = A list of stations that can be used in the game
+        {
+            "station_name1" = The name of the station
+            {
+                "assets" = A list of images for the station. Currently, only one image is supported
+                {
+                    "default" = The default image for the station
+                }
+            }
+        }
+    }
+}
+```
+
+### RobotouilleRenderer and RobotouilleCanvas
+
+The ```RobotouilleRenderer``` class reads the configuration file and creates a ```RobotouilleCanvas```. The configuration file is passed as a string and set as field in ```RobotouilleCanvas``` to be used when drawing images for the player, floor, items, and stations. 
```

Comment:
The RobotouilleRenderer actually sets up the Pygame window and uses RobotouilleCanvas to handle draw images onto the window.